bundles/org.dataflowanalysis.analysis/src/org/dataflowanalysis/analysis/core/AbstractTransposedFlowGraph.java

```diff
@@ -10,29 +10,29 @@
 import java.util.stream.Stream;
 
 /**
- * This abstract class represents a partial flow graph induced by a sink {@link AbstractPartialFlowGraph#getSink()}.
- * Ambiguous flows, like multiple flows to one input pin, are resolved in the partial flow graph and represent one
+ * This abstract class represents a transposed flow graph induced by a sink {@link AbstractTransposedFlowGraph#getSink()}.
```

Comment:
It's called [transpose](https://en.wikipedia.org/wiki/Transpose_graph) flow graph, not `transposed` (so no `d`) at the end, see #137. I also missed this when starting to work with the new terminology but it seems that there actually is no `d` at the end. Easy fix: Full search on replace on the code base for `transposed` -> `transpose` :)

Reply:
Fixed in 5657b74977446d7c52d2b772a36f42aeced03d65